applications/notifications/server/notifications/services/notification/adapters/email_adapter.py

```diff
@@ -0,0 +1,23 @@
+from cloudharness.utils.config import CloudharnessConfig as conf
+from notifications.services.notification.adapters.base_adapter import NotificationBaseAdapter
+
+
+class NotificationEmailAdapter(NotificationBaseAdapter):
+    def __init__(self, *args, **kwargs):
```

Comment:
some comment to explain the scope of this adapter would help. The crucial information would be about how this is configured and where the templates are taken from

Reply:
done